### PR TITLE
Feature/pet registry

### PR DESCRIPTION
**Pull request by rodrigofmeneses**

### CODE REVIEW

File: tests/src/main/routes/pet-routes.test.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  })
+
+  return body
+}
+
+describe('POST - /api/pet Route', () => {
+  it.each([
+    ['Cachorro', undefined, { status: 201, specie: { name: 'Cachorro' }, specieAlias: null }],
+    ['Outros', 'Inseto', { status: 201, specie: { name: 'Outros' }, specieAlias: 'Inseto' }]
+    // ['Cachorro', 'Gato', { status: 201, specie: { name: 'Gato' }, specieAlias: null }], // TODO
```
Comment (andersonnaz) [inline tests/src/main/routes/pet-routes.test.ts]: remove comments.

File: tests/src/main/routes/pet-routes.test.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    // ['Outros', undefined, { status: 201, specie: { name: 'Gato' }, specieAlias: null }] // TODO
```
Comment (andersonnaz) [inline tests/src/main/routes/pet-routes.test.ts]: remove comments.

**Review by andersonnaz — DISMISSED** [review]

**Review by andersonnaz — DISMISSED** [review]

File: src/data/protocols/db/specie/load-species-repository.ts
Comment (matheusgondra) [inline src/data/protocols/db/specie/load-species-repository.ts:1]: why is there an empty file?

**Review by matheusgondra — DISMISSED** [review]

File: src/data/protocols/db/specie/load-species-repository.ts
Comment (rodrigofmeneses) [reply src/data/protocols/db/specie/load-species-repository.ts:1]: Nice, it will be removed.

**Review by andersonnaz — APPROVED** [review]

**Review by matheusgondra — APPROVED** [review]

**Pull request merged**